nestkernel/node.h

```diff
@@ -477,6 +477,59 @@ class Node
   virtual void handle( DoubleDataEvent& e );
 
   /**
+   * return the Ca_minus value at t (in ms).
+   * return 0.0 if not overridden
+   */
+  virtual double_t
```

Comment:
Right now, I cannot see where all these getter functions are used. Inspection of the node should be via `get_status()`. Unless there is a very good reason to have these getters, I strongly suggest to remove them all.